models/User.php

```diff
@@ -127,6 +127,19 @@ public function getIsAdmin()
     }
 
     /**
+     * @return mixed return the assigned authorization items of the current user.
+     * @throws \InvalidArgumentException when auth manager is not configured.
+     */
+    public function getAuthItems()
```

Comment:
I think in this case its okay to return an empty array when there is no manager.
Since if there is no manager then by definition there are no authorization items for the current (or any) user.

This would also fix the crash in your view.